cmd/insprctl/cli/apply_test.go

```diff
@@ -338,7 +338,7 @@ func Test_applyValidFiles(t *testing.T) {
 			},
 			want:    nil,
 			funcErr: ierrors.NewError().Message("unauthorized").Unauthorized().Build(),
-			errMsg:  "We couldn't authenticate with the cluster. Is your token configured correctly?\n",
+			errMsg:  "we couldn't authenticate with the cluster. Is your token configured correctly?\n",
```

Comment:
'We' is suggestive of an action performed by a group, I would use ' Failure to authenticate ...'

Reply:
You're right